lib/pure/collections/lists.nim

```diff
@@ -22,50 +22,43 @@
 ##
 ## Lists
 ## -----
-##
-## .. code-block::
-##   import lists
-##
-##   var
-##     l = initDoublyLinkedList[int]()
-##     a = newDoublyLinkedNode[int](3)
-##     b = newDoublyLinkedNode[int](7)
-##     c = newDoublyLinkedNode[int](9)
-##
-##   l.append(a)
-##   l.append(b)
-##   l.prepend(c)
-##
-##   assert a.next == b
-##   assert a.prev == c
-##   assert c.next == a
-##   assert c.next.next == b
-##   assert c.prev == nil
-##   assert b.next == nil
-##
-##
+runnableExamples:
```

Comment:
I know you already fixed this, but the issue was not the `runnableExamples` itself, but a missing blank line, this would have worked:
```nim
## Lists
## -----
##
runnableExamples:
  ...
```

Reply:
not really, that still had issues

Reply:
indeed, tracked in https://github.com/nim-lang/Nim/issues/16990